packages/sdk-react/src/components/approvalPolicies/useApprovalPolicyTrigger.test.tsx

```
@@ -16,6 +16,12 @@ import {
 import { tagListFixture } from '@/mocks/tags';
 import { renderHook } from '@testing-library/react';
 
+import { OPERATOR_OPERATIONS, type OperatorOperation } from './triggerUtils';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'OPERATOR_OPERATIONS'. Consider to delete it or rename it.

'OPERATOR_OPERATIONS' is defined here:


(lint/suspicious/noRedeclare)